a10_octavia/api/drivers/driver.py

```diff
@@ -304,3 +319,37 @@ def validate_flavor(self, flavor_dict):
                                   'due to: {}'.format(str(e)),
                 operator_fault_string='Failed to validate the flavor metadata '
                                       'due to: {}'.format(str(e)))
+
+    # Availability Zone
+    def get_supported_availability_zone_metadata(self):
```

Comment:
We need to raise a NotImplemented() here. The current information would imply to the operator that they have their availability zones misconfigured. Not that our driver doesn't support this operation.

Reply:
Per https://docs.openstack.org/octavia/latest/contributor/guides/providers.html
> **get_supported_availability_zone_metadata¶**
> Retrieves a dictionary of supported availability zone keys and their description.

So, I think we should return empty dictionary for we support nothing. And if we raise exception command ```openstack loadbalancer provider capability list a10``` will failed with 500 Server Internal Error.